portfolio/src/main/webapp/assets/icons/secret.svg

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"

```

Comment:
It seems like the dom elements aren't indented properly. I assume this is copy pasta from somewhere?

Reply:
Yeah this is just an svg I downloaded and added to my assets folder. I did not write any of this code haha.